### PR TITLE
Update uefi and fm-ucode to new revision; Fix the weston-init issue

### PR DESCRIPTION
**Pull request by junzhuimx**
Update to Q4 2022 revision

### CODE REVIEW

**Review by otavio — APPROVED** [review]

File: recipes-dpaa/fm-ucode/fm-ucode_git.bb [hunk 1]
```diff
@@ -1,14 +1,14 @@
 DESCRIPTION = "Fman microcode binary"
 SECTION = "fm-ucode"
-LICENSE = "Freescale-Binary-EULA"
-LIC_FILES_CHKSUM = "file://Freescale-Binary-EULA;md5=b784c031868ba1bd5ebc5de372c823fa"
+LICENSE = "NXP-Binary-EULA"
+LIC_FILES_CHKSUM = "file://NXP-Binary-EULA.txt;md5=12e248d404ce1ea8bed0148fcf127e69"
```
Comment (angolini) [inline recipes-dpaa/fm-ucode/fm-ucode_git.bb:4]: I would add a note in the commit log addressing the license change.

**Review by angolini — APPROVED** [review]

File: recipes-graphics/wayland/weston-init.bbappend [hunk 2]
```diff
@@ -24,11 +24,12 @@ INI_UNCOMMENT_ASSIGNMENTS:append:mx8mq-nxp-bsp = " \
 # FIXME: The 8QM and 8QXP SoCs have better performance without G2D so don't enable it
 # Ideally, this should be seamless and Vivante ought to handle it internally and take the fastest
 # rendering code.
+INI_UNCOMMENT_USE_G2D               ?= ""
 INI_UNCOMMENT_USE_G2D:imxgpu2d      ?= "use-g2d=1"
 INI_UNCOMMENT_USE_G2D:mx8qm-nxp-bsp  = ""
 INI_UNCOMMENT_USE_G2D:mx8qxp-nxp-bsp = ""
 INI_UNCOMMENT_USE_G2D:mx8dx-nxp-bsp  = ""
```
Comment (thochstein) [inline recipes-graphics/wayland/weston-init.bbappend:31]: All of these should have weak assignment `?=`.

File: recipes-graphics/wayland/weston-init.bbappend [hunk 3]
```diff
@@ -24,11 +24,12 @@ INI_UNCOMMENT_ASSIGNMENTS:append:mx8mq-nxp-bsp = " \
...
-INI_UNCOMMENT_ASSIGNMENTS:append:imxgpu2d = " \
+INI_UNCOMMENT_ASSIGNMENTS:append = " \
```
Comment (thochstein) [inline recipes-graphics/wayland/weston-init.bbappend:32]: I'm unclear about this. So far we don't add `imxgpu2d` for i.MX 93 https://github.com/nxp-imx/meta-imx/blob/kirkstone-5.15.71-2.2.0/meta-bsp/conf/machine/include/imx-base-extend.inc#L32. The part has 2D, though I think it is different from other 2D implementaitons we support.

Comment (otavio) [reply recipes-graphics/wayland/weston-init.bbappend:32]: I am confused now. The imx93 has a different 2D GPU?

Comment (junzhuimx) [reply recipes-graphics/wayland/weston-init.bbappend:32]: i.MX93 has no GPU hardware, but a different PXP G2D hardware.

Comment (thochstein) [reply recipes-graphics/wayland/weston-init.bbappend:32]: @junzhuimx thanks for the clarification.

**Review by otavio — APPROVED** [review]

**Pull request merged**